src/services/anchor-service.ts

```diff
@@ -361,19 +366,25 @@ export class AnchorService {
     const candidates = merkleTree.getLeaves()
     const anchors = []
 
+    // Simple performance timing 
+    const timings = []
+   
     for (let i = 0; i < candidates.length; i++) {
       const candidate = candidates[i]
       logger.debug(
         `Creating anchor commit #${i + 1} of ${
           candidates.length
         }: stream id ${candidate.streamId.toString()} at commit CID ${candidate.cid}`
       )
+      let before = Date.now()
       const anchor = await this._createAnchorCommit(candidate, i, ipfsProofCid, merkleTree)
       if (anchor) {
         anchors.push(anchor)
+        timings.push(Date.now() - before)
       }
     }
 
+    Metrics.recordAverage(METRIC_NAMES.TIME_ANCHOR_COMMITS_MS, timings)
```

Comment:
I am a little confused to why we are timing this

Reply:
yeah will do a separate PR with timings and traces